src/arch/arm/fastmodel/FastModel.py

```diff
@@ -75,21 +75,21 @@ def __init__(self, width, desc):
         super(VectorAmbaInitiatorSocket, self).__init__(
                 my_role, desc, is_source=True)
 
-class ScMasterPort(Port):
```

Comment:
Let's back out this change. This is changing something quite different than everything else in this commit.

Reply:
So should I keep it as a separate commit?
Or undo the changes?

Reply:
Sorry for the slow reply here :). Probably back out... this is one that we might have to do something special for